internal/provisioner/kops_provisioner.go

```diff
@@ -224,11 +225,52 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster) error
 		}
 	}
 
+	// Setup Helm
+	err = HelmSetup(logger, kops)
+	if err != nil {
+		return err
+	}
+
 	logger.WithField("name", kopsMetadata.Name).Info("Successfully created cluster")
 
 	return nil
 }
 
+// HelmSetup is used for the initial setup of Helm in cluster
+func HelmSetup(logger log.FieldLogger, kops *kops.Cmd) error {
+	logger.Infof("Initializing Helm in the cluster")
+	init := exec.Command("helm", "--kubeconfig", kops.GetKubeConfigPath(), "init", "--upgrade")
+	err := init.Run()
+	if err != nil {
+		return err
+	}
+	logger.Infof("Creating service account")
+	svcacc := exec.Command("kubectl", "--kubeconfig", kops.GetKubeConfigPath(), "--namespace", "kube-system", "create", "serviceaccount", "tiller")
```

Comment:
We have a builtin Go library for interfacing with kubernetes resources. We should be using that instead of exec'ing `kubectl`.

https://github.com/mattermost/mattermost-cloud/tree/master/internal/tools/k8s